scripts/babel-plugin-vite-define.cjs

```
@@ -0,0 +1,148 @@
+let viteDefines = null
+
+/**
+ * Load Vite config and extract define replacements
+ */
+async function loadViteDefines() {
+  if (viteDefines !== null) {
+    return viteDefines
+  }
+
+  try {
+    const configFile = path.resolve(__dirname, '../vite.config.mts')
```

ESLint

[error] '__dirname' is not defined.

(no-undef)